assets/scripts/async-submit.js

```diff
@@ -57,7 +57,4 @@
 
   passphraseInput.focus()
   submitButton.removeAttribute('disabled')
-
-  // Preload font awesome
-  try { document.fonts.load('14px FontAwesome') } catch (e) {}
```

Comment:
It is still used here: https://github.com/cozy/cozy-stack/blob/master/assets/scripts/async-submit.js#L44